cmake/FindCOCAL.cmake

```diff
@@ -0,0 +1,53 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Find the COCAL initial data code (assumed repository or documentation link)
+#
+# Pass `COCAL_ROOT` to the CMake build configuration to set up the following
+# targets:
+#
+# - COCAL::Exporter: Functionality to load COCAL volume data and interpolate to
+#   arbitrary points.
+
+if(NOT DEFINED COCAL_ROOT OR COCAL_ROOT STREQUAL "")
+  set(COCAL_ROOT $ENV{COCAL_ROOT})
+endif()
+
+if (DEFINED COCAL_ROOT AND NOT COCAL_ROOT STREQUAL "")
+  
+  # Find the COCAL library
+  find_library(
+    COCAL_LIB
+    NAMES libcocal.a
+    PATHS ${COCAL_ROOT}/lib
```

Comment:
presumably `HINTS` would be closer to what the others do. Note that `lib` is system specific (could be `lib` or `lib64` or `lib/x86_64-linux-gnu` to name a few).